src/main/java/io/reactivex/internal/schedulers/ComputationScheduler.java

```diff
@@ -96,6 +97,25 @@ public void shutdown() {
                 w.dispose();
             }
         }
+
+        @Override
+        public void createWorkers(int number, WorkerCallback callback) {
+            int c = cores;
```

Comment:
`cores` is `final` in this context, why copy it to local var?

Reply:
With potential volatiles around, this won't re-read the field every time it is needed in the loop below.